src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -247,6 +269,14 @@ private AdditionalSettingsFormSegment buildFormAdditionalSettings(
         
         additionalSettings.setResponseVisibleNeverChecked((newFeedbackSession != null  
                                                             && Const.TIME_REPRESENTS_NEVER.equals(newFeedbackSession.resultsVisibleFromTime)));
+        
+        additionalSettings.setSendClosingEmailChecked(newFeedbackSession == null 
+                                                      || newFeedbackSession.isClosingEmailEnabled);
+        additionalSettings.setSendOpeningEmailChecked(newFeedbackSession == null 
+                                                      || newFeedbackSession.isOpeningEmailEnabled);
+        additionalSettings.setSendPublishedEmailChecked(newFeedbackSession == null 
+                                                        || newFeedbackSession.isPublishedEmailEnabled);
+        
```

Comment:
This isn't fully related to this PR, but there are 10 null checks for `newFeedbackSession` in this method alone.
Using a `if-else` would definitely be more tidy than having a ternary operator every other line.